frontend/src/components/button/followbutton/followbutton.module.scss

```diff
@@ -0,0 +1,20 @@
+@use '~/src/styles/theme';
+@import '~/src/styles/theme';
+
+.followButton {
+  // Had to use !important tag as these properties were overridden by default css of Button MUI
```

Comment:
This is interesting - just took a look and it seems like using !important has a "use as a last resort" attached to it (?) - do none of these options work on it? https://mui.com/customization/how-to-customize/

Reply:
Ok so here is the problem. The classes and global overriding, I tried it but doesn't work, (used .Mui-Button-root) but the button remains the same. 

Now the things that worked for me are: !important and [Reusable styles overrides.](https://mui.com/customization/how-to-customize/#2-reusable-style-overrides). 
The problem with reusable styles and inline styling, I cannot use the `theme.scss` provided by the team and had to use hex or rgb codes. 

Hope this makes sense